server/views/referrals/show/otherReferrals.njk

```diff
@@ -0,0 +1,62 @@
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "../../partials/keylessSummaryCard.njk" import keylessSummaryCard %}
+
+{% extends "./partials/referralLayout.njk" %}
+
+{% block primaryContent %}
+
+    {% if tableRows | length  %}
+    <p> {{ person.name }} has other open referrals.<p>
+    {{ govukTable({
+        attributes: {
+            "data-module": "moj-sortable-table"
+        },
+
+        head: [
+            {
+                text: "Programme",
+                attributes: {
+                    "aria-sort": "ascending"
+                }
+            },
+            {
+                text: "Strand",
+                attributes: {
+                    "aria-sort": "none"
+                }
+            },
+            {
+                text: "Location",
+                attributes: {
+                    "aria-sort": "none"
+                }
+            },
+            {
+                text: "Referrer",
+                attributes: {
+                    "aria-sort": "none"
+                }
+            },
+            {
+                text: "Date referred",
+                attributes: {
+                    "aria-sort": "none"
```

Comment:
May need an extra `"data-sort-value": "..."` attribute, so the date column is sorted correctly.